core/schema/container.graphqls

```diff
@@ -132,9 +132,17 @@ type Container {
 
   "Retrieves this container plus a cache volume mounted at the given path."
   withMountedCache(
+    "Path to mount the cache volume at."
     path: String!
+
+    "ID of the cache to mount."
     cache: CacheID!
+
+    "Directory to use as the cache volume's root."
     source: DirectoryID
+
+    "Sharing mode of the cache volume."
+    sharing: CacheSharingMode
```

Comment:
I think it's possible to express defaults like this:

```suggestion
    sharing: CacheSharingMode=SHARED
```

If that works, seems slightly better for self-documenting.

Reply:
I've been trying it out, doesn't seem to work: 
<img width="611" alt="image" src="https://user-images.githubusercontent.com/31691250/217515107-59454a2f-2574-45ac-8b50-67d716c7caa8.png">
<img width="589" alt="image" src="https://user-images.githubusercontent.com/31691250/217515168-983788cf-7459-4fa0-9baf-9b3471ff8225.png">
<img width="416" alt="image" src="https://user-images.githubusercontent.com/31691250/217515208-ecee1f11-f276-424b-b11c-c7fcd30ed605.png">